pkg/ingester/ingester_test.go

```diff
@@ -6839,7 +6850,7 @@ func TestIngester_UpdateLabelSetMetrics(t *testing.T) {
 	require.NoError(t, os.Mkdir(chunksDir, os.ModePerm))
 	require.NoError(t, os.Mkdir(blocksDir, os.ModePerm))
 
-	i, err := prepareIngesterWithBlocksStorageAndLimits(t, cfg, limits, tenantLimits, blocksDir, reg, false)
+	i, err := prepareIngesterWithBlocksStorageAndLimits(t, cfg, limits, tenantLimits, blocksDir, reg)
```

Comment:
Do we have a test case for when native histogram is disabled and push should increment the samples discarded metrics?
If not, can you add one to `TestIngester_Push`?

Reply:
https://github.com/cortexproject/cortex/blob/master/pkg/ingester/ingester_test.go#L1034